src/services/YotiService.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-console */
 import { Logger } from "@aws-lambda-powertools/logger";
-import crypto, { randomUUID }from "crypto";
+import crypto, { randomUUID } from "crypto";
```

Comment:
are both needed?

Reply:
Yes crypto is used for getRSASignatureForMessage at line 51